frontend/js/store/modules/config.js

```diff
@@ -0,0 +1,17 @@
+const state = {
+  publishDate24Hr: window[process.env.VUE_APP_NAME].STORE.config.publishDate24Hr,
```

Comment:
It doesn't look like this isn't being used in Vue components, so we should be able to remove it.